WorkloadManagementSystem/Client/DownloadInputData.py

```diff
@@ -294,15 +299,23 @@ def __downloadFromSE( self, lfn, seName, reps, guid ):
     localFile = os.path.join( downloadDir, fileName )
     result = StorageElement( seName ).getFile( lfn, localPath = downloadDir )
     if not result['OK']:
-      self.log.warn( 'Problem getting %s at %s:\n%s' % ( lfn, seName, result['Message'] ) )
+      self.log.warn( 'Problem getting %s from %s:\n%s' % ( lfn, seName, result['Message'] ) )
       return result
     if lfn in result['Value']['Failed']:
-      self.log.warn( 'Problem getting %s at %s:\n%s' % ( lfn, seName, result['Value']['Failed'][lfn] ) )
+      self.log.warn( 'Problem getting %s from %s:\n%s' % ( lfn, seName, result['Value']['Failed'][lfn] ) )
       return S_ERROR( result['Value']['Failed'][lfn] )
+    if lfn not in result['Value']['Successful']:
```

Comment:
Did you see this happening ? THen there is a bug in the SE plugin, I guess, worth an issue